src/store/reducer.js

```diff
@@ -1,15 +1,21 @@
-import { createReducer, createAction } from '@reduxjs/toolkit';
-export const chooseState = createAction('choose_state');
-
 const initialState = {
-    stateCount: 0,
+  news: [],
+  loading: false,
+  error: '',
```

Comment:
in case of setting error as empty string you need to check it obviouse everywhere: `error !== ''`